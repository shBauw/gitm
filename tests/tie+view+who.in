place A1
place A2
place A3
place A4
place A5
place A6
place A7
view
place A8
place A9
place A10
place A11
place A12
place A13
place A14
place A15
place A16
place A17
place A18
place A19
place B1
place B2
place B3
place B4
place B5
place B6
place B7
place B8
place B9
place B10
place B11
place B12
place B13
place B14
place B15
place B16
place B17
place B18
place B19
place C2
place C1
place C4
place C3
place C6
place C5
place C8
place C7
place C10
place C9
place C12
place C11
place C14
place C13
place C16
place C15
place C18
place C17
place C19
place D2
place D1
place D4
place D3
place D6
place D5
place D8
place D7
place D10
place D9
place D12
place D11
place D14
place D13
place D16
place D15
place D18
place D17
place D19
place E1
place E2
place E3
place E4
place E5
place E6
place E7
place E8
place E9
place E10
place E11
place E12
place E13
place E14
place E15
place E16
place E17
place E18
place E19
place F1
place F2
place F3
place F4
place F5
place F6
place F7
place F8
place F9
place F10
place F11
place F12
place F13
place F14
place F15
place F16
place F17
place F18
place F19
place G2
place G1
place G4
place G3
place G6
place G5
place G8
place G7
place G10
place G9
place G12
place G11
place G14
place G13
place G16
place G15
place G18
place G17
place G19
place H2
place H1
place H4
place H3
place H6
place H5
place H8
place H7
place H10
place H9
place H12
view
place H11
place H14
place H13
place H16
place H15
place H18
place H17
place H19
place I1
place I2
place I3
place I4
place I5
place I6
place I7
place I8
place I9
place I10
place I11
place I12
place I13
place I14
place I15
place I16
place I17
place I18
place I19
place J1
place J2
place J3
place J4
place J5
place J6
place J7
place J8
place J9
place J10
place J11
place J12
place J13
place J14
place J15
place J16
place J17
place J18
place J19
place K2
place K1
place K4
place K3
place K6
place K5
place K8
place K7
place K10
place K9
place K12
place K11
place K14
place K13
place K16
place K15
place K18
place K17
place K19
place L2
place L1
place L4
place L3
place L6
place L5
place L8
place L7
place L10
place L9
place L12
place L11
place L14
place L13
place L16
place L15
place L18
place L17
place L19
place M1
place M2
place M3
place M4
place M5
place M6
place M7
place M8
place M9
place M10
place M11
place M12
place M13
place M14
place M15
place M16
place M17
place M18
place M19
place N1
place N2
place N3
place N4
place N5
place N6
place N7
place N8
place N9
place N10
place N11
place N12
place N13
place N14
place N15
place N16
place N17
place N18
place N19
place O2
place O1
place O4
place O3
place O6
place O5
place O8
place O7
place O10
place O9
place O12
place O11
place O14
place O13
place O16
place O15
place O18
place O17
place O19
place P2
place P1
place P4
place P3
place P6
place P5
place P8
place P7
place P10
place P9
place P12
place P11
place P14
place P13
place P16
place P15
place P18
place P17
place P19
place Q1
place Q2
place Q3
place Q4
place Q5
place Q6
place Q7
place Q8
place Q9
place Q10
place Q11
place Q12
place Q13
place Q14
place Q15
place Q16
place Q17
place Q18
place Q19
place R1
place R2
place R3
place R4
place R5
place R6
place R7
place R8
place R9
place R10
place R11
place R12
place R13
place R14
place R15
place R16
place R17
place R18
place R19
place S2
place S1
place S4
place S3
place S6
place S5
place S8
place S7
place S10
place S9
view
who
place S12
view
who
place S11
place S14
place S13
place S16
place S15
place S18
place S17
place S19
