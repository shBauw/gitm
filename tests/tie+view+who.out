M17,xxxxxxx..........................................
G8,##oo...oo##o..##oo#..oo##o..##oo#..oo##o..##oo#..
E16,o#o#o#o##oo##ooo##oo###oo##ooo##oo###oo##ooo##oo#
B
E8,oo##oo###oo##ooo##oo###oo##ooo##oo###oo##ooo##oo#
W
Wow, a tie!
A1A2A3A4A5A6A7A8A9A10A11A12A13A14A15A16A17A18A19B1B2B3B4B5B6B7B8B9B10B11B12B13B14B15B16B17B18B19C2C1C4C3C6C5C8C7C10C9C12C11C14C13C16C15C18C17C19D2D1D4D3D6D5D8D7D10D9D12D11D14D13D16D15D18D17D19E1E2E3E4E5E6E7E8E9E10E11E12E13E14E15E16E17E18E19F1F2F3F4F5F6F7F8F9F10F11F12F13F14F15F16F17F18F19G2G1G4G3G6G5G8G7G10G9G12G11G14G13G16G15G18G17G19H2H1H4H3H6H5H8H7H10H9H12H11H14H13H16H15H18H17H19I1I2I3I4I5I6I7I8I9I10I11I12I13I14I15I16I17I18I19J1J2J3J4J5J6J7J8J9J10J11J12J13J14J15J16J17J18J19K2K1K4K3K6K5K8K7K10K9K12K11K14K13K16K15K18K17K19L2L1L4L3L6L5L8L7L10L9L12L11L14L13L16L15L18L17L19M1M2M3M4M5M6M7M8M9M10M11M12M13M14M15M16M17M18M19N1N2N3N4N5N6N7N8N9N10N11N12N13N14N15N16N17N18N19O2O1O4O3O6O5O8O7O10O9O12O11O14O13O16O15O18O17O19P2P1P4P3P6P5P8P7P10P9P12P11P14P13P16P15P18P17P19Q1Q2Q3Q4Q5Q6Q7Q8Q9Q10Q11Q12Q13Q14Q15Q16Q17Q18Q19R1R2R3R4R5R6R7R8R9R10R11R12R13R14R15R16R17R18R19S2S1S4S3S6S5S8S7S10S9S12S11S14S13S16S15S18S17S19
Thank you for playing!
